django_readers/rest_framework.py

```diff
@@ -231,22 +244,28 @@ def serializer_class_for_view(view):
     return serializer_class_for_spec(name_prefix, model, view.spec)
 
 
-class PairWithOutAttribute(tuple):
-    out = None
+class PairWithAnnotation(tuple):
+    _readers_annotation = None
 
 
-class StringWithOutAttribute(str):
-    out = None
+class StringWithAnnotation(str):
+    _readers_annotation = None
 
 
-def out(field_or_dict):
-    if isinstance(field_or_dict, dict):
-        if not all(
-            isinstance(item, serializers.Field) for item in field_or_dict.values()
-        ):
-            raise TypeError("Each value must be an instance of Field")
-    elif not isinstance(field_or_dict, serializers.Field):
-        raise TypeError("Must be an instance of Field")
+def out(*args, **kwargs):
+    if args:
+        if len(args) != 1:
+            raise TypeError("Provide a single field or dictionary of fields")
```

Comment:
Here, it may be worth checking that `kwargs` are empty too. 🤔

Reply:
I've decided to explicitly support passing _both_ a field, and also kwargs. See the tests for some examples. I know it's a redundant thing to do (you could just as easily pass the kwargs into the field constructor directly), but it felt more consistent and less surprising to have it "just work"...

Reply:
Great. No problem 👍